src/puppet-web/firer.ts

```diff
@@ -316,7 +316,7 @@ async function checkRoomLeave(m: Message): Promise<void> {
   /**
    * FIXME: leaver maybe is a list
    */
-  let leaverContact = room.member(leaver)
+  let leaverContact = room.member({remark: leaver}) || room.member({nick: leaver})
```

Comment:
> We can keep room.member(inviteeList[i]) here, after you follow the review for member() parameters suggestion.

I don't agree with this suggestion, because of the following reason:

We keep function `room.member()` work, but it should work only with the function find member by nickName.    
We cannot use it both find remark and nickName, because one can set a remark same as nick, then it would be confused. 

So, what do you think about it?